integration-tests/pom.xml

```diff
@@ -36,12 +34,26 @@
 			<build>
 				<plugins>
 					<plugin>
+						<groupId>io.fabric8</groupId>
+						<artifactId>fabric8-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<goals>
+									<goal>resource</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<profile>aggregate</profile>
+						</configuration>
+					</plugin>
+					<plugin>
 						<groupId>org.apache.maven.plugins</groupId>
 						<artifactId>maven-failsafe-plugin</artifactId>
 						<configuration>
 							<systemPropertyVariables>
-								<nameServiceTemplate>${project.parent.basedir}/name-service/target/classes/META-INF/fabric8/openshift.yml</nameServiceTemplate>
-								<greetingServiceTemplate>${project.parent.basedir}/greeting-service/target/classes/META-INF/fabric8/openshift.yml</greetingServiceTemplate>
+								<nameServiceIS>${project.parent.basedir}/name-service/target/name-service-is.yml</nameServiceIS>
+								<greetingServiceIS>${project.parent.basedir}/greeting-service/target/greeting-service-is.yml</greetingServiceIS>
```

Comment:
This is an interesting approach. In Swarm, I did something slightly different, maybe worth trying: https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker/blob/master/tests/pom.xml#L98-L128

Reply:
Thanks @Ladicek , it works with this change.